components/local-app/cmd/root.go

```diff
@@ -55,11 +55,29 @@ var rootCmd = &cobra.Command{
 		var noColor bool
 		if !isatty.IsTerminal(os.Stdout.Fd()) {
 			noColor = true
+			color.Disable()
 		}
 		slog.SetDefault(slog.New(tint.NewHandler(os.Stdout, &tint.Options{
 			Level:      level,
 			NoColor:    noColor,
 			TimeFormat: time.StampMilli,
+			ReplaceAttr: func(groups []string, attr slog.Attr) slog.Attr {
+				if attr.Key != "level" {
+					return attr
+				}
+
+				switch attr.Value.String() {
+				case slog.LevelDebug.String():
+					attr.Value = slog.StringValue(color.Gray.Render("[DEBUG]"))
+				case slog.LevelInfo.String():
+					attr.Value = slog.StringValue(color.Green.Render("[INFO ]"))
```

Comment:
Good that we're keeping the indentation of the messages consistent with this, although it may be a bit confusing for grepping output

Reply:
It also looks a bit odd if there isn't any output that actually aligns here.
Let's run with it for now and change it if it's too odd.